lib/God/ForkMode.js

```diff
@@ -93,7 +93,6 @@ module.exports = function ForkMode(God) {
       try {
         var cspr = spawn(command, args, {
           env      : pm2_env,
-          detached : true,
```

Comment:
do you really need to remove detached here?

Reply:
@soyuka I can't make the windowsHide option works with `detached: true`. Do we actually needs detached to be true here? I maybe mistook something here, but child processes in cluster mode are attached to the daemon process I think.

Reply:
@soyuka I tried to reproduce the problem (`detached:true` with 'windowsHide:true`) to find the real cause, but found something unexpected. I think the console windows may be not caused by this spawn call at all, but something else. I'll do some digging.